lib/danger/danger_core/environment_manager.rb

```diff
@@ -22,57 +22,54 @@ def initialize(env)
 
       raise "Could not find a CI source".red unless self.ci_source
 
-      # only GitHub for now, open for PRs adding more!
-      self.request_source = GitHub.new(self.ci_source, ENV)
-      # Also Git only for now, also open for PRs adding more!
-      self.scm = GitRepo.new # For now
+      RequestSources.constants.each do |symb|
```

Comment:
I think we use this in a few places @justMaku - you're welcome to not and I'll wrap them all up